src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2957,18 +2957,17 @@ namespace WaterToAirHeatPumpSimple {
         // Save component design water volumetric flow rate.
         // Use 1/2 flow since both cooling and heating coil will save flow yet only 1 will operate at a time
         if (simpleWatertoAirHP.RatedWaterVolFlowRate > 0.0) {
-            if (simpleWatertoAirHP.WAHPType == WatertoAirHP::Heating) {
-                PlantUtilities::RegisterPlantCompDesignFlow(
-                    state, simpleWatertoAirHP.WaterInletNodeNum, 0.5 * simpleWatertoAirHP.RatedWaterVolFlowRate);
-                if (simpleWatertoAirHP.CompanionCoolingCoilNum > 0) {
-                    auto &companionCoolingCoil(state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum));
+            if (simpleWatertoAirHP.CompanionCoolingCoilNum > 0) {
+                auto &companionCoolingCoil(state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(simpleWatertoAirHP.CompanionCoolingCoilNum));
+                if (companionCoolingCoil.RatedWaterVolFlowRate > simpleWatertoAirHP.RatedWaterVolFlowRate) {
+                    simpleWatertoAirHP.RatedWaterVolFlowRate = companionCoolingCoil.RatedWaterVolFlowRate;
+                } else {
+                    companionCoolingCoil.RatedWaterVolFlowRate = simpleWatertoAirHP.RatedWaterVolFlowRate;
                     PlantUtilities::RegisterPlantCompDesignFlow(
                         state, companionCoolingCoil.WaterInletNodeNum, 0.5 * simpleWatertoAirHP.RatedWaterVolFlowRate);
                 }
-            } else if (simpleWatertoAirHP.WAHPType == WatertoAirHP::Cooling) {
-                PlantUtilities::RegisterPlantCompDesignFlow(
-                    state, simpleWatertoAirHP.WaterInletNodeNum, 0.5 * simpleWatertoAirHP.RatedWaterVolFlowRate);
             }
+            PlantUtilities::RegisterPlantCompDesignFlow(state, simpleWatertoAirHP.WaterInletNodeNum, 0.5 * simpleWatertoAirHP.RatedWaterVolFlowRate);
```

Comment:
This routine is called sequentially for each coil, so set the flow rate of each coil based on the larger of the two companion coils.

Reply:
This is off topic but if the Sizing:Plant object uses different delta T's shouldn't the flow rate in each operating mode be different (i.e., to actually operate at the Sizing:Plant delta T specified)? This is of course outside the scope of this issue but I think it's important to think of things like that. I guess it would be good to check manufacturers data to see if different water flow rates are ever possible or if 2-speed pumps are ever used. Using the same flow rate for cooling and heating does seem like the right choice. Although I do wonder if there are times when the first coil would post it's result to the eio before the companion coil got a chance to update the water flow rate?

Reply:
I could very well be wrong but I don't think that this issue could arise. Stepping through the debugger I see that, when we have companion coils, because the cooling coil is always evaluated first and its final capacity is only fully determined after the heating coil has been sized, I think that the process to output the flow to the EIO file is only done then (meaning after both coils have been sized) and only once.

On a different note, I see comments in the block of code above this one related to the system capacity used to determined the flow rate. I think these comments are valid: we're currently using the evaporator capacity but we should use the condenser capacity. Should a different PR be created to address these?

Reply:
I would create a different PR for changing the condenser flow sizing method.

Reply:
I think this is worthy of slight pressure on the green button below.